bluebottle/organizations/models.py

```diff
@@ -96,33 +51,12 @@ class OrganizationContact(models.Model):
     name = models.TextField(_('name'), max_length=100)
     email = models.EmailField(_('email'), max_length=254)
     phone = models.TextField(_('phone'), max_length=40)
-    owner = models.ForeignKey(settings.AUTH_USER_MODEL, verbose_name=_('creator'))
-    organization = models.ForeignKey('organizations.Organization',
-                                     related_name="contacts")
+    owner = models.ForeignKey(settings.AUTH_USER_MODEL, verbose_name=_('owner'))
+
```

Comment:
So an `OrganizationContact` doesn't have a relation to an `Organization` anymore?

Reply:
Maybe later (or rather sooner) Organizations could also be Partners that also have some rights to create or change projects, where multiple OrganizationContacts/Members are able to login and edit the project that Organization is partnering with.